maraboupy/MarabouNetworkNNet.py

```diff
@@ -141,18 +285,108 @@ def read_nnet(self, file_name):
             self.weights = weights
             self.biases = biases
 
-            # Convert input bounds from their original values to normalized values if normalization is not being incorporated
             if not self.normalize:
                 for i in range(self.inputSize):
                     self.inputMinimums[i] = (self.inputMinimums[i] - self.inputMeans[i]) / self.inputRanges[i]
                     self.inputMaximums[i] = (self.inputMaximums[i] - self.inputMeans[i]) / self.inputRanges[i]
 
+
+
+    def writeNNet(self, file_name: str):
+        """Write network data into the .nnet file format
+
+        Args:
+            file_name (str): File where the network will be written
+
+        :meta private:
+        """
+
+        # Open the file we wish to write
+        with open(file_name, 'w') as f2:
+
+            #####################
+            # First, we write the header lines:
+            # The first line written is just a line of text
+            # The second line gives the four values:
+            #     Number of fully connected layers in the network
+            #     Number of inputs to the network
+            #     Number of outputs from the network
+            #     Maximum size of any hidden layer
+            # The third line gives the sizes of each layer, including the input and output layers
+            # The fourth line gives an outdated flag, so this can be ignored
+            # The fifth line specifies the minimum values each input can take
+            # The sixth line specifies the maximum values each input can take
+            #     Inputs passed to the network are truncated to be between this range
+            # The seventh line gives the mean value of each input and of all outputs
+            # The eighth line gives the range of each input and of all outputs
+            #     These two lines are used to map raw inputs to the 0 mean, unit range of the inputs and outputs
+            #     used during training
+            # The ninth line begins the network weights and biases
+            ####################
+            # f2.write("// The contents of this file are licensed under the Creative Commons\n")
```

Comment:
Likewise, please remove commented-out code

Reply:
actually, I wasn't sure whether these lines should or should not be in the header of a file created by writing an nnet into a file, what do you think?

Reply:
I don't care much either way - but we should either uncomment or remove